client/src/lib/src/types/BitbakeScanResult.ts

```diff
@@ -38,6 +38,7 @@ export interface BitbakeScanResult {
   _includes: ElementInfo[]
   _recipes: ElementInfo[]
   _overrides: string[]
+  _confFiles: ElementInfo[]
```

Comment:
I'm not sure I understand why you need to list the conf files in this PR. bitbake will give you the full paths, you shouldn't need to search for them on the disk.

Reply:
ok it's for getDefinitionForDirectives(). You should instead refactor that function to also work with the proper URLs provided by bitbake when available.

Reply:
they are very important. I really think you should not use the files array from the scanner for the list of includes when you can get the URLs directly from bitbake -e.

Reply:
Will get the conf files through the command output as addressed in another conversation. Closing this one.